2023-04-30 19:40:28,811 [main] INFO com.classes.BaseTest - Loading all methods from -masterVO class
2023-04-30 19:40:28,819 [main] INFO com.utils._masterVO - Loading Properties file from: .//src//test//resources//sysconfig//Constant.properties
2023-04-30 19:40:28,826 [main] INFO com.utils._masterVO - Properties File Loaded successfully
2023-04-30 19:40:28,829 [main] INFO com.utils._masterVO - Loading Master Sheet from: src//test//resources//config//DataProvider.xlsx
2023-04-30 19:40:30,248 [main] INFO com.utils._masterVO - Master Sheet Loaded successfully
2023-04-30 19:40:30,254 [main] INFO com.utils._masterVO - Loading Client Library for Client Versioning from: .//src//test//resources//sysconfig//ClientLib.properties
2023-04-30 19:40:30,273 [main] INFO com.utils._masterVO - Client Library Loaded successfully
2023-04-30 19:40:30,681 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::ZO - AUTbKlI - AUToKa1C
2023-04-30 19:40:30,888 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::AR - AUT7dba - AUT63dCN
2023-04-30 19:40:31,145 [main] INFO com.utils._masterVO - loadGeoDomains Map returns ::SA - AUTO2Fm - AUTNelnh
2023-04-30 19:40:31,938 [main] INFO com.utils._masterVO - Loading test case file file from: .//src//test//resources//sysconfig//testCaseFile.properties
2023-04-30 19:40:32,013 [main] INFO com.utils._masterVO - Test cases file Loaded successfully
2023-04-30 19:40:32,016 [main] INFO com.utils._masterVO - Loading Web Cases Master file from: .//src//test//resources//config//testCaseFile.properties
2023-04-30 19:40:32,047 [main] INFO com.utils._masterVO - Gateway Cases Master File Loaded successfully
2023-04-30 19:40:32,101 [main] INFO com.dbrepository.DBHandler - Entered Access Handler :: PostGreSQL
2023-04-30 19:40:32,110 [main] INFO com.commons.CacheController - New Cache DAO Loaded :: UPDATECACHE_VER = 1
2023-04-30 19:40:32,179 [MLog-Init-Reporter] INFO com.mchange.v2.log.MLog - MLog clients using log4j logging.
2023-04-30 19:40:32,383 [main] INFO com.mchange.v2.c3p0.C3P0Registry - Initializing c3p0-0.9.5.2 [built 08-December-2015 22:06:04 -0800; debug? true; trace: 10]
2023-04-30 19:40:32,519 [main] INFO com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource - Initializing c3p0 pool... com.mchange.v2.c3p0.ComboPooledDataSource [ acquireIncrement -> 5, acquireRetryAttempts -> 30, acquireRetryDelay -> 1000, autoCommitOnClose -> false, automaticTestTable -> null, breakAfterAcquireFailure -> false, checkoutTimeout -> 0, connectionCustomizerClassName -> null, connectionTesterClassName -> com.mchange.v2.c3p0.impl.DefaultConnectionTester, contextClassLoaderSource -> caller, dataSourceName -> 1hge0z1av1memv2511n881d|3932c79a, debugUnreturnedConnectionStackTraces -> false, description -> null, driverClass -> org.postgresql.Driver, extensions -> {}, factoryClassLocation -> null, forceIgnoreUnresolvedTransactions -> false, forceSynchronousCheckins -> false, forceUseNamedDriverClass -> false, identityToken -> 1hge0z1av1memv2511n881d|3932c79a, idleConnectionTestPeriod -> 0, initialPoolSize -> 10, jdbcUrl -> jdbc:postgresql://172.30.38.120:5432/pretups_trunk_dev, maxAdministrativeTaskTime -> 0, maxConnectionAge -> 0, maxIdleTime -> 0, maxIdleTimeExcessConnections -> 0, maxPoolSize -> 20, maxStatements -> 0, maxStatementsPerConnection -> 0, minPoolSize -> 10, numHelperThreads -> 3, preferredTestQuery -> null, privilegeSpawnedThreads -> false, properties -> {user=******, password=******}, propertyCycle -> 0, statementCacheNumDeferredCloseThreads -> 0, testConnectionOnCheckin -> false, testConnectionOnCheckout -> false, unreturnedConnectionTimeout -> 0, userOverrides -> {}, usesTraditionalReflectiveProxies -> false ]
2023-04-30 19:40:33,398 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getNetworkName()
2023-04-30 19:40:33,517 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: Network Name (Network GGN)
2023-04-30 19:40:33,519 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(AMOUNT_MULT_FACTOR)
2023-04-30 19:40:33,521 [main] INFO com.dbrepository.DBConnectionPool - Active Size:1 Cache Size:2
2023-04-30 19:40:33,608 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (100)
2023-04-30 19:40:33,611 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(MSISDN_PREFIX_LENGTH)
2023-04-30 19:40:33,614 [main] INFO com.dbrepository.DBConnectionPool - Active Size:2 Cache Size:1
2023-04-30 19:40:33,706 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (2)
2023-04-30 19:40:33,708 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(MAX_MSISDN_LENGTH)
2023-04-30 19:40:33,710 [main] INFO com.dbrepository.DBConnectionPool - Active Size:3 Cache Size:0
2023-04-30 19:40:33,967 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (15)
2023-04-30 19:40:33,969 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(USER_PRODUCT_MULTIPLE_WALLET)
2023-04-30 19:40:33,971 [main] INFO com.dbrepository.DBConnectionPool - Active Size:4 Cache Size:2
2023-04-30 19:40:34,083 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (false)
2023-04-30 19:40:34,085 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(EXTERNAL_DATE_FORMAT)
2023-04-30 19:40:34,087 [main] INFO com.dbrepository.DBConnectionPool - Active Size:5 Cache Size:1
2023-04-30 19:40:34,200 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yyyy)
2023-04-30 19:40:34,202 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DATE_FORMAT_CAL_JAVA)
2023-04-30 19:40:34,204 [main] INFO com.dbrepository.DBConnectionPool - Active Size:6 Cache Size:0
2023-04-30 19:40:34,526 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (dd/MM/yyyy)
2023-04-30 19:40:34,529 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(OTH_COM_CHNL)
2023-04-30 19:40:34,531 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2023-04-30 19:40:34,650 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (false)
2023-04-30 19:40:34,652 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DW_ALLOWED_GATEWAYS)
2023-04-30 19:40:34,654 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2023-04-30 19:40:34,881 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (EXTGW,DWEXTGW)
2023-04-30 19:40:34,883 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DEFAULT_LANGUAGE)
2023-04-30 19:40:34,885 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2023-04-30 19:40:35,022 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (en)
2023-04-30 19:40:35,024 [main] INFO com.dbrepository.PostGreSQLRepository - Entered :: getSystemPreference(DEFAULT_COUNTRY)
2023-04-30 19:40:35,026 [main] INFO com.dbrepository.DBConnectionPool - Active Size:0 Cache Size:9
2023-04-30 19:40:35,160 [main] INFO com.dbrepository.PostGreSQLRepository - Query Repository Returns: DEFAULT_VALUE (US)
2023-04-30 19:40:35,162 [main] INFO com.utils._masterVO - Display Locale set to: en_US
2023-04-30 19:40:41,383 [main] INFO com.sshmanager.SSHService - Exiting getFileFromSFTP()
2023-04-30 19:40:43,385 [main] INFO com.sshmanager.SSHService - Exiting getFileFromSFTP()
2023-04-30 19:40:43,387 [main] INFO com.utils._masterVO - Loading Messages Resource properties File from: .//src//test//resources//sysconfig//English.properties
2023-04-30 19:40:43,462 [main] INFO com.utils._masterVO - Messages Resource Properties file Loaded successfully
2023-04-30 19:40:43,464 [main] INFO com.utils._masterVO - Loading Messages Resource properties File from: .//src//test//resources//sysconfig//C2S_English.properties
2023-04-30 19:40:43,494 [main] INFO com.utils._masterVO - Messages Resource Properties file Loaded successfully
2023-04-30 19:40:43,609 [main] INFO com.sshmanager.SSHService - Executing: grep '^[^#]' /data1/pretupsapp/tomcat9_pgdev/conf/pretups/LogConfig.props|grep '^log4j.appender.OneLineTXNLogC2S.File[ =]'|cut -d'=' -f2|sed 's/^[ 	]*//;s/[ 	]*$//'
2023-04-30 19:40:43,894 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
2023-04-30 19:40:43,898 [main] INFO com.sshmanager.SSHService - Executing: grep '^[^#]' /data1/pretupsapp/tomcat9_pgdev/conf/pretups/LogConfig.props|grep '^log4j.appender.ChannelRequestDailyLog.File[ =]'|cut -d'=' -f2|sed 's/^[ 	]*//;s/[ 	]*$//'
2023-04-30 19:40:44,448 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
2023-04-30 19:40:44,450 [main] INFO com.classes.BaseTest - Completed loading all methods from -masterVO class
2023-04-30 19:42:23,966 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:23,966 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:23,967 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:23,967 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:23,994 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For A_01_Test_Success -------------------
2023-04-30 19:42:23,994 [main] INFO restassuredapi.test.O2CChannelUserList - Entered A_01_Test_Success()
2023-04-30 19:42:23,996 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2023-04-30 19:42:23,997 [main] INFO restassuredapi.test.O2CChannelUserList - Entered Test_OAuthenticationTest()
2023-04-30 19:42:25,312 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2023-04-30 19:42:26,866 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":null,"refreshToken":null,"status":401,"messageCode":"error.general.processing","message":null,"errorMap":null}
2023-04-30 19:42:27,014 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [200] but found [401]
2023-04-30 19:42:27,242 [main] INFO com.sshmanager.SSHService - Executing: tail -n 1000 /data1/pretupsapp/tomcat9_pgdev/logs/catalina.out

2023-04-30 19:42:27,863 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
2023-04-30 19:42:29,909 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For A_01_Test_Success -------------------
2023-04-30 19:42:29,909 [main] INFO restassuredapi.test.O2CChannelUserList - Entered A_01_Test_Success()
2023-04-30 19:42:29,911 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2023-04-30 19:42:29,912 [main] INFO restassuredapi.test.O2CChannelUserList - Entered Test_OAuthenticationTest()
2023-04-30 19:42:29,918 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2023-04-30 19:42:30,117 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":null,"refreshToken":null,"status":401,"messageCode":"5005","message":"Details of user qa_chnl_adm not found","errorMap":null}
2023-04-30 19:42:30,125 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [200] but found [401]
2023-04-30 19:42:30,191 [main] INFO com.sshmanager.SSHService - Executing: tail -n 1000 /data1/pretupsapp/tomcat9_pgdev/logs/catalina.out

2023-04-30 19:42:30,575 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
2023-04-30 19:42:32,330 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:32,331 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:32,332 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:32,333 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:32,338 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For A_02_Test_Invalid_Domain_Code -------------------
2023-04-30 19:42:32,338 [main] INFO restassuredapi.test.O2CChannelUserList - Entered A_02_Test_Invalid_Domain_Code()
2023-04-30 19:42:32,340 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2023-04-30 19:42:32,340 [main] INFO restassuredapi.test.O2CChannelUserList - Entered Test_OAuthenticationTest()
2023-04-30 19:42:32,348 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2023-04-30 19:42:32,585 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":null,"refreshToken":null,"status":401,"messageCode":"error.general.processing","message":null,"errorMap":null}
2023-04-30 19:42:32,591 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [200] but found [401]
2023-04-30 19:42:32,693 [main] INFO com.sshmanager.SSHService - Executing: tail -n 1000 /data1/pretupsapp/tomcat9_pgdev/logs/catalina.out

2023-04-30 19:42:33,157 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
2023-04-30 19:42:34,553 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For A_02_Test_Invalid_Domain_Code -------------------
2023-04-30 19:42:34,553 [main] INFO restassuredapi.test.O2CChannelUserList - Entered A_02_Test_Invalid_Domain_Code()
2023-04-30 19:42:34,556 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2023-04-30 19:42:34,556 [main] INFO restassuredapi.test.O2CChannelUserList - Entered Test_OAuthenticationTest()
2023-04-30 19:42:34,561 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2023-04-30 19:42:34,942 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":null,"refreshToken":null,"status":401,"messageCode":"5005","message":"Details of user qa_chnl_adm not found","errorMap":null}
2023-04-30 19:42:34,949 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [200] but found [401]
2023-04-30 19:42:35,023 [main] INFO com.sshmanager.SSHService - Executing: tail -n 1000 /data1/pretupsapp/tomcat9_pgdev/logs/catalina.out

2023-04-30 19:42:35,433 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
2023-04-30 19:42:36,969 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:55)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:36,969 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:56)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:36,970 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:36,971 [main] INFO com.utils.ExcelUtility - java.lang.NullPointerException
	at com.utils.ExcelUtility.getCellData(ExcelUtility.java:335)
	at restassuredapi.test.O2CChannelUserList.TestDataFeed(O2CChannelUserList.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)

2023-04-30 19:42:36,978 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For A_03_Test_Invalid_ChannelOwnerCategory -------------------
2023-04-30 19:42:36,978 [main] INFO restassuredapi.test.O2CChannelUserList - Entered A_03_Test_Invalid_ChannelOwnerCategory()
2023-04-30 19:42:36,982 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2023-04-30 19:42:36,985 [main] INFO restassuredapi.test.O2CChannelUserList - Entered Test_OAuthenticationTest()
2023-04-30 19:42:36,991 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2023-04-30 19:42:37,909 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":null,"refreshToken":null,"status":401,"messageCode":"error.general.processing","message":null,"errorMap":null}
2023-04-30 19:42:37,915 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [200] but found [401]
2023-04-30 19:42:37,990 [main] INFO com.sshmanager.SSHService - Executing: tail -n 1000 /data1/pretupsapp/tomcat9_pgdev/logs/catalina.out

2023-04-30 19:42:38,811 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
2023-04-30 19:42:40,108 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For A_03_Test_Invalid_ChannelOwnerCategory -------------------
2023-04-30 19:42:40,108 [main] INFO restassuredapi.test.O2CChannelUserList - Entered A_03_Test_Invalid_ChannelOwnerCategory()
2023-04-30 19:42:40,111 [main] INFO restassuredapi.test.O2CChannelUserList - ------------------ Test Case Execution Started For Test_OAuthenticationTest -------------------
2023-04-30 19:42:40,111 [main] INFO restassuredapi.test.O2CChannelUserList - Entered Test_OAuthenticationTest()
2023-04-30 19:42:40,117 [main] INFO restassuredapi.api.BaseAPI - API URL: http://172.30.38.232:6789/pretups/rstapi/v1/generateTokenAPI
2023-04-30 19:42:40,483 [main] INFO restassuredapi.api.oauthentication.OAuthenticationAPI - {"token":null,"refreshToken":null,"status":401,"messageCode":"5005","message":"Details of user qa_chnl_adm not found","errorMap":null}
2023-04-30 19:42:40,490 [main] INFO com.classes.BaseTest - java.lang.AssertionError: expected [200] but found [401]
2023-04-30 19:42:40,744 [main] INFO com.sshmanager.SSHService - Executing: tail -n 1000 /data1/pretupsapp/tomcat9_pgdev/logs/catalina.out

2023-04-30 19:42:41,731 [main] INFO com.sshmanager.SSHService - Channel Exit Status: 0
